### DISCUSSION TITLE
Thermostating in MD in DFTB+

### DISCUSSION THREAD

**Discussion by achyuttt** [dftbplus/dftbplus#1473] [General]
Hello everyone,

I am trying to perform MD simulation in DFTB+ for a NVT ensemble using NoseHoover Thermostat. 
My input files look like this:
I want to equilibrate my system at 900k but when I tracked the temperature of the system, it reduces to minimum upto 200K and maximum upto 1000K as well. 

What could be the probable reason for this temperature variation despite using a Thermostat?

Thermostat =  NoseHoover{
    Temperature [Kelvin] = 900.0
	AdaptFillingTemp = Yes
	CouplingStrength [cm^-1] = 3200
  }

**Comment by aradi** [comment]
It takes time, until the thermostat thermalizes the system. I guess, your initial positions / velocities are far away from one which would be a likely configuration at 900 Kelvin.

**Reply by achyuttt** [reply]
Thank you for your response Dr. Balint. 
Does the value of CouplingStrength effect the result? How does one set its value? Is it hit and trial  or we need to find it prior to our MD calculations? if its hit and trial what factor do we need to consider to get the good CouplingStrength value for our calculation?

**Reply by aradi** [reply]
You're value seems to be OK to me. Systems do need time to thermalize. If coupling strength is too strong, your system won't be able to evolve according its internal forces (as the thermostat will permanently overrule the forces...)
